### PR TITLE
SC2Client: Changes to /download_data and feedback

### PR DESCRIPTION
**Pull request by SoldierofOrder**
## What is this fixing or adding?
Old users who have never run the /download_data command in SC2Client do not have a version number for the maps on their system. This PR warns these users that their maps are probably outdated, and that they should run `/download_data`. This should cut back on people having issues with the Safe Haven and Haven's Fall missions.

Also, `/download_data` no longer accepts a parameter called `force`, and will now *always* force a download when run. This is to simplify the user interface; we did have an issue already where someone needed to force a download but didn't understand how to do it.

## How was this tested?
- Deleted version number file on a Windows 10 machine that already had the most recent map files
- Tried to connect to a server (the correct feedback appeared)
- Ran `/download_data` twice with no arguments (the client did the download both times)

### CODE REVIEW

**Comment by Magnemania** [comment]
Downloaded and tested it.  Looks good to me, but we will need @TheCondor07 's approval as the active maintainer.

**Comment by TheCondor07** [comment]
Tested. Works good on my end.

File: Starcraft2Client.py
Comment (Berserker66) [inline Starcraft2Client.py:206]: this feels like a .warning() level?

**Review by Berserker66 — APPROVED** [review]

**Pull request merged**
